[DOC] Updated TraceQL doc with an example for arrays (#4661)
grafana/tempo: docs/sources/tempo/release-notes/v2-7.md

{{< admonition type="note" >}}
This release disables gRPC compression by default to improve performance.

> This change may increase data usage and network traffic, which can impact Grafana Cloud billing, especially for larger clusters. Refer to [gRPC compression](#grpc-compression-disabled) for more information.

Author: ```suggestion
```

> This change may increase data usage and network traffic, which can impact Grafana Cloud billing, especially for larger clusters. Refer to [gRPC compression](#grpc-compression-disabled) for more information.

Author: ```suggestion
This change will increase data usage and network traffic, which can impact cloud billing depending on your configuration. Refer to [gRPC compression](#grpc-compression-disabled) for more information.
```

Our benchmark suggests that without compression, queriers and distributors use less CPU and memory.

{{< admonition type="note" >}}

> While disabling gRPC compression improves performance, it will increase data usage and network traffic, which can impact cloud billing, especially for larger clusters.

Author: ```suggestion
While disabling gRPC compression improves performance, it will increase data usage and network traffic, which can impact cloud billing depending on your configuration.
```

> While disabling gRPC compression improves performance, it may increase data usage and network traffic, which can impact Grafana Cloud billing, especially for larger clusters.

Author: ```suggestion
While disabling gRPC compression improves performance, it will increase data usage and network traffic, which can impact cloud billing, especially for larger clusters.
```
